app/enterprise/0.36-x/deployment/installation/docker.md

```diff
@@ -1,146 +1,229 @@
-key, which will be used in step 3. Alternatively, to retrieve it from
-Bintray, click <a href="https://bintray.com/profile/edit" target="_blank">here</a>.
+To complete this guide you will need:
 
-3. For **users with existing contracts**, add the Kong Docker repository and
-pull the image:
+- Docker
+- [Bintray](https://bintray.com) credentials (Your **Sales** or **Support** 
+contact will email your Bintray credential to you.)
+- A valid Kong Enterprise license file (accessed via Bintray)
+
+
+## Step 1. Download Kong Enterprise
+
+1. Obtain your Bintray API Key
+
+    Log in to [Bintray](https://bintray.com)
+
+    Hover over your user name in the top navigation bar and select "Edit Profile"
+    from the dropdown.
+
+    _[insert screenshot]_
+
+    From the Profile page, select "API Key" from the sidebar.
+
+    _[insert screenshot]_
+
+    Submit your Bintray password and copy your API key to your clipboard.
+
+    _[insert screenshot]_
+
+2. Log in to Docker and pull the image
 
     ```
-    $ docker login -u <your_username_from_bintray> -p <your_apikey_from_bintray> kong-docker-kong-enterprise-edition-docker.bintray.io
+    $ docker login -u <bintray_username> -p <bintray_API_key>
+    kong-docker-kong-enterprise-edition-docker.bintray.io
+
     $ docker pull kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
     ```
 
-   For **trial users**, run the following, replacing `<your trial image URL>`
-with the URL you received in your welcome email:
+    This will pull the latest edition of Kong Enterprise. To pull a specific
+    version of Kong Enterprise add the `<VERSION>` to the url:
 
     ```
-    curl -Lsv "<your trial image URL>" -o /tmp/kong-docker-ee.tar.gz
-    docker load -i /tmp/kong-docker-ee.tar.gz
+    $ docker pull kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition/<VERSION>
     ```
 
-4. You should now have your Kong Enterprise image locally. Run
-`docker images` to verify and find the image ID matching your repository. 
+3. Run `docker images` to find the ID for the Kong Enterprise
+image.
 
-5. Tag the image ID for easier use in the commands that follow:
+4. Tag the image ID for easier use in the commands that follow:
 
     ```
-    docker tag <IMAGE ID> kong-ee
+    $ docker tag <IMAGE_ID> kong-ee
     ```
 
-    (Replace "IMAGE ID" with the one matching your repository, seen in step 4)
-
-6. Create a Docker network
+5. Create a Docker network (_optional_)
 
-    You will need to create a custom network to allow the containers to discover
-    and communicate with each other. In this example, `kong-ee-net` is the network name,
-    but you can use any name.
+    Containers require a network in order to discover and communicate with each
+    other. If you would like to use this functionality, you can create a network
+    using the following command, replaying `kong-ee-net` with whatever you would
+    like to name your network:
```

Comment:
Containers require a network in order to discover and communicate with each other. To use this functionality create a network using the following command, replacing `kong-ee-net` with the name of your network: